packages/loot-core/src/client/state-types/app.d.ts

```diff
@@ -14,6 +14,7 @@ export type AppState = {
   managerHasInitialized: boolean;
   lastUndoState: { current: UndoState | null };
   lastSplitState: { current: SplitState | null };
+  budgetMonth?: unknown;
```

Comment:
Turns out this was missing from the interface

Reply:
It also looks to be totally unused so it can probably just be removed.

Reply:
Very true: https://github.com/actualbudget/actual/pull/1427/commits/4c43018f5148838850152d7e84e7df4c0e17896a